local/network.go

```diff
@@ -130,33 +132,46 @@ func init() {
 		panic(err)
 	}
 
-	startTime := time.Now().Unix()
-	lockTime := startTime + genesisLocktimeStartimeDelta
-	genesisMap["startTime"] = float64(startTime)
-	allocations, ok := genesisMap["allocations"].([]interface{})
-	if !ok {
-		panic(errors.New("could not get allocations in genesis"))
-	}
-	for _, allocIntf := range allocations {
-		alloc, ok := allocIntf.(map[string]interface{})
+	if _, ok := os.LookupEnv("CAMINO_NETWORK"); ok {
+		camino, ok := genesisMap["camino"].(map[string]interface{})
 		if !ok {
-			panic(fmt.Errorf("unexpected type for allocation in genesis. got %T", allocIntf))
+			panic(errors.New("could not get camino"))
 		}
-		unlockSchedule, ok := alloc["unlockSchedule"].([]interface{})
+		genesisMap["networkID"] = constants.KopernikusID
+		genesisMap["initialStakeDuration"] = float64(0)
+		genesisMap["initialStakedFunds"] = []string{}
+		genesisMap["initialStakers"] = []genesis.UnparsedStaker{}
+		genesisMap["allocations"] = []genesis.UnparsedAllocation{}
+		camino["verifyNodeSignature"] = true
+		camino["lockModeBondDeposit"] = true
+	} else {
+		startTime := time.Now().Unix()
```

Comment:
We want this, too for camino.
It's a cool thing, and we recently changed that offerID's do not chnage anymore if StartTime changes

Reply:
done